### ISSUE TITLE
Improved Video Quality For Enlarged View

### ISSUE THREAD

**Issue by eudisd** [VIDY/embed.js#186]
higher quality video in enlarged view and in desktop view would go a long way if we can get those in (480 is fine)

**Comment by eudisd** [comment]
Hey @lukeed I think you mentioned that you've been working on this right?  I can't remember

**Comment by lukeed** [comment]
Yes, can't be done need. Needs housekeeping in the file buckets. These legacy videos have larger sizes but new items don't/won't.

**Comment by lukeed** [comment]
Reopened because a larger video/thumbnail is causing the jitteriness again, undoing the effects of #212
